packages/amplify_api_plugin/amplify_api_android/android/src/main/kotlin/com/amazonaws/amplify/amplify_api/Api.kt

```diff
@@ -40,7 +40,7 @@ import kotlinx.coroutines.CoroutineDispatcher
 import kotlinx.coroutines.Dispatchers
 
 /** AmplifyApiPlugin */
-class AmplifyApiPlugin : FlutterPlugin, MethodCallHandler {
+class Api : FlutterPlugin, MethodCallHandler {
```

Comment:
I kinda prefer it before - what was the reason for this change?

Reply:
To be consistent with existing modules.  For Auth, Analytics, Storage we have adopted the same naming scheme, ie. AuthCognito / AnalyticsPinpoint / StorageS3

It's just a bit weirder here because API doesn't really have a specific AWS tie in.